Objects/moduleobject.c

```diff
@@ -866,6 +866,40 @@ _Py_module_getattro(PyModuleObject *m, PyObject *name)
     return _Py_module_getattro_impl(m, name, 0);
 }
 
+static int
+module_setattro(PyModuleObject *m, PyObject *name, PyObject *value)
```

Comment:
Can you rename "m" to "mod"? I dislike single letter variable names.

Reply:
BTW, this was for consistency with the rest of module (see functions above and below).